lib/dry/transaction/step.rb

```diff
@@ -7,6 +7,7 @@ module Transaction
     # @api private
     class Step
       UNDEFINED = Object.new.freeze
+      NOOP = :itself.to_proc.freeze
```

Comment:
I notice this same `NOOP` proc is called `LOOPBACK` inside `Stack`. Is the difference in naming intentional?

Reply:
Yeah, sort of, I'd say they serve different purposes. In fact, `NOOP` is only needed for unit tests and we could pass it there instead